app/workers/find_and_delete_scheduled_accounts_worker.rb

```diff
@@ -4,6 +4,8 @@ class FindAndDeleteScheduledAccountsWorker
   include Sidekiq::Worker
 
   def perform
+    # TODO: remove this condition before 2.6
```

Comment:
I wouldn't add this comment. Right now no one imagines we could keep this longer than that, but we never know.
I think a Jira should be enough to remember.

Reply:
Makes sense @guicassolato, also, we hide the nasty details hehehe

Reply:
The Jira issue [THREESCALE-2204 - Fix efficiency to destroy tenants or services](https://issues.jboss.org/browse/THREESCALE-2204) has just been created now 😉